docs/planejamento/metodologias.md

```diff
@@ -0,0 +1,112 @@
+<br>
+
+* <b>Coragem:</b> Um dos valores mais importantes é ter coragem para testar novas ideias, substituir antigas, mudar o código corrigir erros.
+ <br>
+
+* <b>Respeito:</b> Por último, mas não menos importante, ter respeito pela comunidade ao seu redor, pelos membros do grupo e pelo meio social, para manter um ambiente agradável para todos trabalharem.
+ <br>
+
+#### <a>Práticas do XP</a>
+<p style="text-indent: 20px; text-align: justify">
+O XP, além dos valores listados acima, possui um conjunto de princípios que devem ser seguidos para que se tenha uma boa prática de desenvolvimento no projeto. Abaixo, foram listadas algumas práticas escolhidas pelo nosso grupo para serem utilizadas. 
+</p>
+
+
+* <b>Programação em par:</b> Prática de programar a dois, assim, com duas pessoas olhando o mesmo código fonte, a chance desse mesmo código conter bugs e falhas é diminuída pela metade. Ela é uma das práticas primordiais do XP, pois dois programadores fazendo o trabalho juntos acaba agregando muito para o trabalho em equipe.
+
+* <b>Ritmo sustentável:</b> Manter um ritmo de trabalho constante com qualidade, onde eles estejam atentos e dispostos.
+
+* <b>Releases curtas:</b> As liberações de pequenas versões funcionais do projeto auxiliam muito no processo de aceitação por parte do cliente que já pode testar uma parte do sistema. Além de que a chance de reverter os erros/bugs do sistema ficam mais fáceis de consertar por voltar para uma versão mais perto da destinada.
+
+
+## <a>Srum</a>
+<p style="text-indent: 20px; text-align: justify">
+Scrum é uma metodologia ágil de gerenciamento de projetos que visa aumentar a eficiência, a transparência e a colaboração em equipes de desenvolvimento de software. Foi criada para lidar com projetos complexos e em constante mudança, permitindo a entrega de valor de forma incremental e iterativa.
+</p>
+
+<p style="text-indent: 20px; text-align: justify">
+A metodologia Scrum é baseada em três pilares principais: transparência, inspeção e adaptação. A transparência envolve compartilhar informações relevantes com todos os membros da equipe para garantir uma compreensão comum do projeto e suas metas. A inspeção refere-se à avaliação contínua do progresso e do produto resultante, permitindo que a equipe identifique e corrija problemas ou ajuste o plano conforme necessário. A adaptação envolve a capacidade de ajustar e otimizar o trabalho e o processo à medida que a equipe aprende mais sobre o projeto e suas necessidades.
+</p>
+
+#### <a>Papéis no Scrum</a>
+<p style="text-indent: 20px; text-align: justify">
+Promovendo a transparência, a inspeção e a adaptação contínuas, permitindo que as equipes entreguem valor de forma eficiente e eficaz, adaptando-se às mudanças e às necessidades do projeto. A metodologia Scrum define três papéis principais:
```

Comment:
Acredito que a primeira parte do texto fiquei melhor estruturado desta maneira:

Promovendo transparência, inspeção e adaptação contínuas, o Scrum permite que as equipes entreguem valor de maneira eficiente e eficaz, adaptando-se às mudanças e necessidades do projeto.

E acredito que esse texto que ele deveria ser colocado como parágrafo final do texto acima, pois ele se conecta melhor com aquela parte. 
Fazendo com que o tópico 'Papéis no Scrum', tenha como parágrafo apenas:

A metodologia Scrum define três papéis principais: